app/models/requirement_template.rb

```diff
@@ -0,0 +1,14 @@
+class RequirementTemplate < ApplicationRecord
+  belongs_to :activity, class_name: "Activity"
+  belongs_to :permit_type, class_name: "PermitType"
```

Comment:
does the classname need to be specified? Asking because we have something similar in productmotion where TreeItem is the base class and Feature and Collection inherit from it. It seems like rails is able to automatically handle this in pm code base, so wondering if the same can be achieved here

Reply:
probably